airflow/www/views.py

```diff
@@ -1237,31 +1237,27 @@ def legacy_code(self):
     @provide_session
     def code(self, dag_id, session=None):
         """Dag Code."""
-        all_errors = ""
-        dag_orm = None
+        dag = get_airflow_app().dag_bag.get_dag(dag_id, session=session)
+        dag_model = DagModel.get_dagmodel(dag_id, session=session)
+        if not dag:
+            flash(f'DAG "{dag_id}" seems to be missing.', "error")
+            return redirect(url_for('Airflow.index'))
```

Comment:
```
        dag = get_airflow_app().dag_bag.get_dag(dag_id, session=session)
        if not dag:
            flash(f'DAG "{dag_id}" seems to be missing.', "error")
            return redirect(url_for('Airflow.index'))
```

curious why we add this portion?  it seems that we didn't previously retrieve the dag object?

Reply:
Yeah, if you look down we were passing the orm dag as both the orm dag and the "normal" dag. This was doing the wrong thing before, even if it did work. That might mean we can simplify further, but that'll be another time.

Reply:
ah i see, thought that was in another method for some reason